experimental/dds/tree2/src/test/tree/anchorSet.spec.ts

```diff
@@ -314,6 +317,40 @@ describe("AnchorSet", () => {
 		applyDelta(makeDelta(insertMark, makePath([rootFieldKey, 0], [fieldFoo, 4])), anchors);
 		log.expect([]);
 	});
+
+	// Simple scenario using just anchorSets to validate if cache implementation of the EditableTree.treeStatus api works.
+	it("AnchorNode cache can be set and retrieved.", () => {
```

Comment:
this is a good test; however, I think you also need to test the utility methods you made that depend on this as the gen count comparison logic can have subtle bugs

Reply:
Added unit tests for utility method `treeStatusFromAnchorCache` and `treeStatusFromDetachedField`